app/src/components/SymbolButton.tsx

```
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Pressable, Text, View, StyleSheet } from 'react-native';
+import { Symbol } from '../../db/models';
```

Biome (1.9.4)

[error] Do not shadow the global "Symbol" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)